src/lib/dataEntities/networks.ts

```diff
@@ -293,6 +309,28 @@ export const l2Networks: L2Networks = {
   },
 }
 
+// L2 networks that are a parent chain to Orbit chains.
+function getL2ParentChains(_l2Networks: L2Networks) {
+  const _parentChains: ParentChains = {}
+
+  Object.keys(_l2Networks).map(chainId => {
+    const network = l2Networks[Number(chainId)]
+
+    if (network.partnerChainIDs && network.partnerChainIDs.length > 0) {
```

Comment:
It shouldn't be the case because our list only has 'child chains' in `partnerChainIDs`. If L2 network's L1 chain partner is specified in `partnerChainID` which is a number, not an array.

Are you concerned that someone would mistakenly add an L1 chain ID to the L2 partnerChainIDs list?

Reply:
I see, terminology is bit confusing, it would be better if we had `parentChain` and `childChains`. But I understand we need to keep backward compatibility so it's ok to keep as is